src/components/Upload/index.jsx

```diff
@@ -96,14 +96,14 @@ class Upload extends React.Component {
       .pushBlob(resource, this.onUploadProgress)
       .then((response) => {
         this.setState({
-          success: response.success,
+          success: true,
           loading: false,
-          fileExists: response.fileExists,
+          fileExists: ! response,
           loaded: 100
         });
         this.props.handleUploadStatus({
           loading: false,
-          success: response.success,
+          success: true,
```

Comment:
@shevron does the response will be change in the SDK or do we need to change this response? because the old return is something like the code below. And probably if we need to change  in `datapub`, we also need to change in `datapub-gdx` and `datapub-nhs` .
``` 
const result = {
      oid: object.oid,
      size: object.size,
      name: resource.descriptor.name,
      success: true,
      fileExists: false,
    }
```

Reply:
I changed the SDK's response type. I thought it was way too complex for what we need (really, we just want to know if the file was uploaded or was existing, and otherwise we can expect an exception). 

If this is a problem, I can revert to the old type but I couldn't find a good reason why it needs to be that complex.

Reply:
I don't think is a problem but after merged these changes, please ping me to test/change it `datapub-gdx` if necessary and ping @kmanaseryan  to test and change it in `datapub-nhs`  if necessary.